### PR TITLE
Recode lda_c_vwn

### PR DESCRIPTION
**Pull request by antoine-levitt**

### CODE REVIEW

File: src/xc/lda_c_vwn.jl [hunk 1]
```diff
@@ -2,14 +2,15 @@
...
-    x11 = 1 / (46593 * x1 * cbrt(π*ρ) + x7 + 323380 * sqrt(π*ρ))
-
-    (  x5*atan(x0*x4/(12500*x1 + 46593*x4)) + 310907*log(x11*x7)/10000000
-     + 76037485627899*log(x11*x4*(25000*x1 + 5249*x4)^2/100000)/78468213432500000)
+    # From https://math.nist.gov/DFTdata/atomdata/node5.html
+    A = 0.0310907
+    x0 = -0.10498
+    b = 3.72744
+    c = 12.9352
+    τ = cbrt(3/(4π*ρ))
```
Comment (mfherbst) [inline src/xc/lda_c_vwn.jl:10]: Could you bring this more in light with the other files, e.g. this is the local Seitz radius `rₛ`.

**Comment by mfherbst** [comment]
Also looks like there is something wrong with the indention. Could you check the whitespace?

**Comment by antoine-levitt** [comment]
what's wrong with it?

**Comment by mfherbst** [comment]
My bad, github must have done something strange with the diff formatting.

**Pull request merged**